docs/docs/troubleshooting.md

```diff
@@ -51,27 +51,44 @@ If you are reviewing these errors in the GitHub Actions tab, the contents of `<b
 
-**New Procedure:**
+#### Option 1: Build Reset UF2 in 'zmk-config'
+
+Find the `build.yaml` file in your `zmk-config` folder and add an additional settings reset build for the board used by your split keyboard. For example assume that we have setup nice!nano v2 for Corne, apppend the following `settings_reset` shield to the `build.yaml` file:
+
+```yml
+include:
+  - board: nice_nano_v2
+    shield: corne_left
+  - board: nice_nano_v2
+    shield: corne_right
+  - board: nice_nano_v2
+    shield: settings_reset
+```
+
+Save the file, commit the changes and push them to GitHub. Download the new firmware zip file build by the latest GitHub Actions job. In it you will find an additional `settings_reset` UF2 image file.
+
+#### Option 2: Download Reset UF2 from ZMK's Workflow
 
 1. [Open the GitHub `Actions` tab and select the `Build` workflow](https://github.com/zmkfirmware/zmk/actions?query=workflow%3ABuild+branch%3Amain+event%3Apush).
-1. Find one of the 'results' for which the core-coverage job was successfully run, indicated by a green checkmark in the core-coverage bubble.
-1. From the next page under "Artifacts", download the `$boardname-settings_reset-zmk` zip file.
-1. Unzip the downloaded file.
-1. Put each half of the split keyboard into bootloader mode
-1. Flash one of the halves of the split with the "settings clear" UF2 image from step 1. Immediately after flashing "settings clear" to the chosen half, immediately put it into bootloader mode
-   to avoid accidental bonding between the halves.
-1. Repeat step 3 with the other half of the split keyboard
-1. Flash the actual image for each half of the split keyboard (e.g `my_board_left.uf2` to the left half, `my_board_right.uf2` to the right half)
+2. Find one of the 'results' for which the core-coverage job was successfully run, indicated by a green checkmark in the core-coverage bubble like the image example below.
+3. From the next page under "Artifacts", download and unzip the `$boardname-settings_reset-zmk` zip file for the UF2 image.
 
 | ![Successful core-coverage Job](../docs/assets/troubleshooting/splitpairing/corecoverage.png) |
 | :-------------------------------------------------------------------------------------------: |
 |  An example of a successful core-coverage job which will produce a settings_reset firmware.   |
 
-After completing these steps, pair the halves of the split keyboard together by resetting them at the same time. Most commonly, this is done by grounding the reset pins
-for each of your keyboard's microcontrollers or pressing the reset buttons at the same time.
+#### Reset Split Keyboard Procedure
+
+Perform the following steps to reset both halves of your split keyboard:
+
+1. Put each half of the split keyboard into bootloader mode.
+2. Flash one of the halves of the split with the downloaded settings reset UF2 image. Immediately after flashing the chosen half, immediately put it into bootloader mode to avoid accidental bonding between the halves.
```

Comment:
```suggestion
2. Flash one of the halves of the split with the downloaded settings reset UF2 image. Immediately after flashing the chosen half, put it into bootloader mode to avoid accidental bonding between the halves.
```